pkg/policy/mapstate.go

```diff
@@ -579,6 +608,9 @@ func (e mapStateEntry) String() string {
 // addKeyWithChanges adds a 'key' with value 'entry' to 'keys' keeping track of incremental changes in 'adds' and 'deletes', and any changed or removed old values in 'old', if not nil.
 func (ms *mapState) addKeyWithChanges(key Key, entry mapStateEntry, changes ChangeState) bool {
 	var datapathEqual bool
+
+	entry.Cookie = ms.generateCookie(entry.derivedFromRules)
```

Comment:
question: Couldn't this move into the second if branch below? Or does the cookie change for an incremental add?

For the merge case you're already recomputing the cookie from what I can tell, so this value ends up unused?